src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ResolvedPackageReference.xaml

```diff
@@ -22,6 +22,9 @@
                     ReadOnly="True"
                     DisplayName="Version"
                     Description="Version of dependency.">
+        <StringProperty.DataSource>
```

Comment:
Is version the only property we need this for?

Reply:
Yes, so far other potentially generated attributes (like NoWarn, PrivateAssets, ExcludeAssets, and IncludeAssets) get added in element style because they tend to be more verbose. But we could revisit that decision in the future